check_kernel

```diff
@@ -99,8 +99,17 @@ def is_redhat():
 
 
 def installed_kernel_versions():
+    """
+    Debian-based kernels come in different flavors.
+
+    On some of them, like Proxmox' PVE kernels, special care has to be taken
+    when selecting the right packages to run `dpkg-query` on.
+    """
     if is_debian():
-        return installed_kernel_versions_debian()
+        return \
+          installed_kernel_versions_debian(pkgname='linux-image') \
+          or installed_kernel_versions_debian(pkgname='pve-kernel',
+                                              pkgquery='pve-kernel*-pve')
```

Comment:
On this section, I want to share some more details.

Because packages with names starting with `pve-kernel-` will also include
a meta package having a totally different version number, the program will
have to account for that.

Example:

    pve-kernel-5.13 7.1-9
    pve-kernel-5.13.19-2-pve 5.13.19-4
    pve-kernel-5.13.19-x-pve 5.13.19-x
    ...

When the first package would be picked up, the program would croak:

    /usr/lib/nagios/plugins/check_kernel
    KERNEL WARNING - running version 5.13.19-14, installed: 7.1-9

By using the package query expression `pve-kernel*-pve`, it was easy to work around this problem.